### ISSUE TITLE
Adding ember-a11y-testing to a client app with this theme did NOT report expected axe violations

### ISSUE THREAD

**Issue by acorncom** [IgnaceMaes/qunit-theme-ember#9]
I just was adding `ember-a11y-testing` into a client app and unexpectedly did NOT get axe violations reported back when there were clear violations in the app. I haven't had time yet to dig into the details, but within axe every test result returned for `ember-a11y-testing` to report on is reporting that it is `inapplicable`:

![image](https://github.com/user-attachments/assets/c1b4774f-a912-4b15-9f63-49d9edf92ca4)

Switching back to the default `qunit` theme in this app end up with the expected axe violations I was expecting.

Haven't yet tracked down of these issues are `qunit-theme-ember` or `ember-a11y-testing` CSS overrides not properly applying to this theme, but wanted to report this in case someone else hits the snag before I get a patch up
